src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -168,6 +170,11 @@ void Run (DirectoryAssemblyResolver res)
 				ErrorOnCustomJavaObject     = ErrorOnCustomJavaObject,
 			};
 
+			Log.LogDebugMessage ("All typemap assemblies:");
+			foreach (string asm in allTypemapAssemblies) {
+				Log.LogDebugMessage ($"  {asm}");
+			}
+
```

Comment:
Do we actually need these log messages or was this for debugging an issue? I've seen apps with 250+ assemblies--it might impact performance for large projects.

Reply:
ah, rats, you're right, forgot to remove it